boards/arm/stm32l562e_dk/doc/index.rst

```diff
@@ -0,0 +1,283 @@
+
+- RTC with HW calendar, alarms and calibration
+- Up to 114 fast I/Os, most 5 V-tolerant, up to 14 I/Os with independent supply down to 1.08 V
+- Up to 22 capacitive sensing channels: support touchkey, linear and rotary touch sensors
+- Up to 16 timers and 2 watchdogs
+
+  - 2x 16-bit advanced motor-control
+  - 2x 32-bit and 5x 16-bit general purpose
+  - 2x 16-bit basic
+  - 3x low-power 16-bit timers (available in Stop mode)
+  - 2x watchdogs
+  - 2x SysTick timer
+
+- Memories
+
+  - Up to 512 MB Flash, 2 banks read-while-write
+  - 512 KB of SRAM including 64 KB with hardware parity check
+  - External memory interface for static memories supporting SRAM, PSRAM, NOR, NAND and FRAM memories
+  - OCTOSPI memory interface
+
+- Rich analog peripherals (independent supply)
+
+  - 3x 12-bit ADC 5 MSPS, up to 16-bit with hardware oversampling, 200 uA/MSPS
+  - 2x 12-bit DAC, low-power sample and hold
+  - 2x operational amplifiers with built-in PGA
+  - 2x ultra-low-power comparators
+  - 4x digital filters for sigma delta modulator
+
+- 19x communication interfaces
+
+  - USB Type-C / USB power delivery controller
+  - 2.0 full-speed crystal less solution, LPM and BCD
+  - 2x SAIs (serial audio interface)
+  - 4x I2C FM+(1 Mbit/s), SMBus/PMBus
+  - 6x USARTs (ISO 7816, LIN, IrDA, modem)
+  - 3x SPIs (7x SPIs with USART and OCTOSPI in SPI mode)
+  - 1xFDCAN
+  - 1xSDMMC interface
+  - 2x 14 channel DMA controllers
+
+- CRC calculation unit
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell |trade|
+
+
+More information about STM32L562QE can be found here:
+
+- `STM32L562QE on www.st.com`_
+- `STM32L562 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr stm32l562e_dk board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
```

Comment:
These 3 don't seem supported yet on this board

Reply:
The documentation has been updated to remove the item that are not yet supported.